backend/engine/processor/sbom.py

```diff
@@ -82,11 +82,47 @@ def get_component(name: str, version: str, scan: Scan, component_type: str = Non
     return component
 
 
+def convert_output(output_str: str):
```

Comment:
should we define this in a central place that can be imported here and in `trivy_sbom/main.py`?